app/gist/new/route.js

```diff
@@ -13,12 +13,16 @@ export default GistRoute.extend({
     var model = this.store.createRecord('gist', {description: 'New Twiddle'});
     model.get('files').pushObject(this.store.createRecord('gistFile', {
       filePath: 'templates/application.hbs',
-      content: '<h1>Welcome to {{appName}}</h1><br><br>',
+      content: '<h1>Welcome to {{appName}}</h1><br><br>{{outlet}}<br><br>',
     }));
     model.get('files').pushObject(this.store.createRecord('gistFile', {
       filePath: 'controllers/application.js',
       content: 'import Ember from \'ember\';\n\nexport default Ember.Controller.extend({\n  appName:\'Ember Twiddle\'\n});',
     }));
+    model.get('files').pushObject(this.store.createRecord('gistFile', {
+      filePath: 'router.js',
+      content: 'import Ember from \'ember\';\nvar Router = Ember.Router.extend({\n  location: \'none\'\n});\n\nRouter.map(function() {\n});\n\nexport default Router;\n'
```

Comment:
I can add an address bar if you'd like, but I'm not sure what you mean by an actual router?